docs/azure_arc_jumpstart/azure_arc_data/microk8s/azure/arm_template/dc_vanilla/_index.md

```diff
@@ -142,35 +148,54 @@ As mentioned, this deployment will leverage ARM templates. You will deploy a sin
   --resource-group Arc-Data-Microk8s \
   --name arcdatademo \
   --template-uri https://raw.githubusercontent.com/microsoft/azure_arc/main/azure_arc_data_jumpstart/microk8s/azure/arm_template/azuredeploy.json \
+  --parameters customLocationRPOID="$customLocationRPOID" \
   --parameters azuredeploy.parameters.json
   --parameters templateBaseUrl="https://raw.githubusercontent.com/your--github--handle/azure_arc/microk8s-data/azure_arc_data_jumpstart/microk8s/azure/arm_template/"
   ```
 
   > **NOTE: The deployment time for this scenario can take ~15-20min**
 
-  ![Deployment time](./02.png)
+  ![Deployment time](./01.png)
```

Comment:
Please add the "Screenshot showing...." prefix **to all screenshots' descriptions** like we have in other places